src/test/resources/db/mysql/V1__base.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE `student_detail`
```

Comment:
can change folder to "prune-reserved" or "pruned" or something similar? Because this file is not related to test the db compatibility

Reply:
@Blackdread - Fixed